packages/peregrine/lib/talons/SignIn/signIn.gql.js

```diff
@@ -0,0 +1,51 @@
+import { gql } from '@apollo/client';
+
+export const GET_CUSTOMER = gql`
+    query GetCustomerAfterSignIn {
+        customer {
+            id
+            email
+            firstname
+            lastname
+            is_subscribed
+        }
+    }
+`;
+
+export const SIGN_IN = gql`
+    mutation signIn($email: String!, $password: String!) {
+        generateCustomerToken(email: $email, password: $password) {
+            token
+        }
+    }
+`;
+
+export const CREATE_CART = gql`
+    mutation CreateCartAfterSignIn {
+        cartId: createEmptyCart
+    }
+`;
+
+export const MERGE_CARTS = gql`
```

Comment:
Same treatment as `createAccount.gql.js`; fragments were removed but components still appear to be re-rendering correctly. Still worth focusing on in testing.